examples/fluid.amp.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+    <title>DoubleClick with Fluid Example</title>
+    <link rel="canonical" href="http://nonblocking.io/">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script>
+    <style amp-custom>
+      .artificialfold {
+        height: 1000px;
+        width: 300px
+        background: #000;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>AMP Static Image DFP Reservation.</h1>
+    <p>Tests a static image ad in an AMP page.</p>
+    <div class="artificialfold"></div>
+    <div style="margin-left: 35%; margin-right: 15%; border: 1px red solid">
+      <amp-ad type=doubleclick data-slot="/30497360/native_v2/iu0/iu1/iu2" height="fluid"></amp-ad>
+    </div>
+    <!--<amp-ad type=doubleclick data-slot="/30497360/native_v2/iu7" height="fluid"></amp-ad>-->
```

Comment:
Did you mean for this to be commented out?

Reply:
Doesn't really matter. I uncommented just to get percy to restart.